addons/mail/static/src/models/attachment/attachment.js

```diff
@@ -319,6 +338,12 @@ registerModel({
         isImage: attr({
             compute: '_computeIsImage',
         }),
+        /**
+         * Determines if the attachement has link preview informations.
+         */
+        isLinkPreview: attr({
```

Comment:
Would rename to `hasLinkPreview`. I don't like "is" because it feels like a type/classification whereas "has" feels more like a characteristic. UI show them in a dedicated category, but it's just a UI/UX decision.

This matches more with the docstring, and the rest of code feels more natural to read.

Reply:
It's literally this, the attachment IS a link preview. It even has a custom mime type that said it is a link preview.
It's not something that is linked to the attachment
If the docstring isn't right, we can update it. Maybe just "States" instead of determines.

Reply:
Using mimetype for telling info that this attachment has a link preview is an implementation detail.
I would say a very bad implementation detail, but I cannot convince people that choose using mimetype for that.
I don't want this bad decision infect the rest of modelling.

Anyway, it's a detail for the moment.

I'm just afraid that this field name motivates considering this as a type/classification, and make assumption in business code about this, and make it hard to fix if we decide that's actually not a type/classification but a trait. As a said, it's used as type/classification right now just from current UI/UX and other modelling consideration at present. Type/classification are maintenance nightmare when they are inappropriately introduced, hence only define them when you really want it.